Added Picker Network Component (#16340)
MetaMask/metamask-extension: ui/components/component-library/picker-network/README.mdx

import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';

import { PickerNetwork } from './picker-network';

# PickerNetwork

> The `PickerNetwork` is a component used to display text within a container.

Reviewer: Could update this description to something like

```suggestion
The `PickerNetwork` is used for the action of changing a network
```
